### DISCUSSION TITLE
[question] schema是否是DSL，schema是针对所有高级语言（这里是vue代码）都适用的吗？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [opentiny/tiny-engine#876] [Q&A] [answered]
在tinyengine中，是有将schema转换为vue代码的包：`@opentiny/tiny-engine-dsl-vue`

请问：

1、是否有这个的功能设计的描述呢？
2、schema是否是DSL，schema是针对所有高级语言（这里是vue代码）都适用的吗？
3、比如，现在想要开发生产其他代码的库（例如：`@opentiny/tiny-engine-dsl-react`， `@opentiny/tiny-engine-dsl-java`），那么是参考哪个文档可以实现？

**Comment by chilingling** [comment, marked as answer]
您可以参考下文档：https://opentiny.design/tiny-engine#/help-center/course/engine/7342098765432108
